bubble-wrap-style.yaml

```diff
@@ -6122,7 +5476,7 @@ layers:
             draw:
                 mapzen_icon_library:
                     sprite: train_station
-                    size: [[13, 12px], [14, 14px], [15, 16px], [17, 20px]]
+                    size: [[13, [65%,65%]], [16, [100%,100%]]]
```

Comment:
See notes above on how this and other %-based scaling below can be simplified to only use a single value for both dimensions

Reply:
ff50c44c36332a80c4f5b015d167f9a34bf033e3